build.py

```diff
@@ -72,12 +72,14 @@
 TRITON_VERSION_MAP = {
     "2.40.0dev": (
         "23.11dev",  # triton container
-        "23.09",  # upstream container
+        "23.10",  # upstream container
         "1.16.0",  # ORT
         "2023.0.0",  # ORT OpenVINO
         "2023.0.0",  # Standalone OpenVINO
         "2.4.7",  # DCGM version
         "py310_23.1.0-1",  # Conda version
+        "9.1.0.1",  # TRT version for building TRT-LLM backend
+        "12.2",  # CUDA version for building TRT-LLM backend
```

Comment:
I think these were added by mistake when resolving merge conflicts. Will need to remove these two line.

Reply:
Addressed